Directory.Build.targets

```diff
@@ -28,14 +28,12 @@
       <IncludePreReleaseLabelInPackageVersion Condition="'$(SuppressFinalPackageVersion)' == 'true'">true</IncludePreReleaseLabelInPackageVersion>
       <IncludePreReleaseLabelInPackageVersion Condition="'$(IsShipping)' != 'true'">true</IncludePreReleaseLabelInPackageVersion>
 
-      <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
-      <IncludeBuildNumberInPackageVersion Condition="'$(SuppressFinalPackageVersion)' == 'true'">true</IncludeBuildNumberInPackageVersion>
-      <IncludeBuildNumberInPackageVersion Condition="'$(IsShipping)' != 'true'">true</IncludeBuildNumberInPackageVersion>
-
       <ProductVersionSuffix Condition="'$(IncludePreReleaseLabelInPackageVersion)' == 'true'">-$(VersionSuffix)</ProductVersionSuffix>
       <ProductBandVersion Condition="'$(ProductBandVersion)' == ''">$(MajorVersion).$(MinorVersion)</ProductBandVersion>
       <ProductionVersion Condition="'$(ProductionVersion)' == ''">$(ProductBandVersion).$(PatchVersion)</ProductionVersion>
       <ProductVersion>$(ProductionVersion)$(ProductVersionSuffix)</ProductVersion>
+      <ProductVersionWithSuffix>$(ProductionVersion)</ProductVersionWithSuffix>
+      <ProductVersionWithSuffix Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">$(ProductionVersion)-$(VersionSuffix)</ProductVersionWithSuffix>
```

Comment:
I'm a little confused this works, I would expect Arcade to unset `VersionSuffix` based on https://github.com/dotnet/arcade/blob/ca919f224b58b17e48d8786dd555ae9a1f6cdfa3/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L103-L112, and this would end up as `3.0.0-`.

Reply:
And I think Arcade would have to introduce a new property to let us do this, since IIRC `VersionSuffix` is used directly by the NuGet pack targets (and that's why Arcade needs to clear it).